drivers/manual_hmis_data/app/views/manual_hmis_data/project_cocs/edit.haml

```diff
@@ -1,5 +1,5 @@
 = content_for :modal_title do
-  Edit Project CoC Record for #{@project_coc.project.name} at #{@project_coc.project.organization.name}
+  Edit Project CoC Record for #{@project_coc.project.name(current_user)} at #{@project_coc.project.organization.name}
```

Comment:
I _think_ you need `can_edit_projects?` to get to this page.  Perhaps we should pass that in as well?

Reply:
updated to ignore confidential status